_includes/delete-token.md

```diff
@@ -11,13 +11,13 @@ No other states are supported.
 **Request**
 
 ```http
-PATCH /psp/{{ api_resource }}/{% unless api_resource == "paymentorders" %}payments/{% endunless %}{{ page.payment_id }} HTTP/1.1
+PATCH /psp/{{ api_resource }}/{% unless api_resource == "paymentorders" %}payments/instrumentdata/{% endunless %}{{ page.payment_id }} HTTP/1.1
```

Comment:
If `instrumentdata` occurs in the URL, I suppose it does so also for `paymentorders`? Is this operation supported on payment orders at all, btw?

It also seems like there's a payment token and not an ID in the URL.

```suggestion
PATCH /psp/{{ api_resource }}/{% unless api_resource == "paymentorders" %}payments/{% endunless %}instrumentdata/{{ page.payment_token }} HTTP/1.1
```

Reply:
I'll check up with Carljohan who reported this!

Reply:
An answer update:
Jag tror jag förstår hans fråga men den måste jag dubbelkolla, för detta handlar ju om anrop för tokens och recur betalningar.